cli/src/commands/services.js

```diff
@@ -27,7 +27,8 @@ export async function dockerComposeUp({ verbose = false }, workingDir, spinner =
     try{
         startSpinner(spinner, 'Starting Botfront...')
         await shellAsync(command, { silent: !verbose });
-        await waitForService('botfront');
+        if (spinner) { await waitForService('botfront'); }
+        else { await wait(20000); process.exit(0); }
```

Comment:
Why don't you just wait for the service to be available?

Reply:
Could not make it run on GCS (tried a bunch).